praw/reddit.py

```diff
@@ -567,11 +568,19 @@ def _objectify_request(
         :param params: The query parameters to add to the request (default:
             None).
         :param path: The path to fetch.
+        :param json: JSON-serializable object to send in the body
+            of the request with a Content-Type header of application/json
+            (default: None). If ``json`` is provided, ``data`` should not be.
```

Comment:
This should be sorted as well. However, I am having difficulty making a suggestion, as there seems to be a cap of 4 lines per suggestion.